workers/generic-worker/loopback_video_linux_test.go

```diff
@@ -0,0 +1,61 @@
+//go:build linux
+
+package main
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/mcuadros/go-defaults"
+)
+
+func TestLoopbackVideo(t *testing.T) {
+	setup(t)
+
+	payload := GenericWorkerPayload{
+		Command: [][]string{
+			{"ls", "-l", "/dev/video0"},
+		},
+		MaxRunTime: 30,
+		Features: FeatureFlags{
+			LoopbackVideo: true,
+		},
+	}
+	defaults.SetDefaults(&payload)
+	td := testTask(t)
+	td.Scopes = append(td.Scopes, "generic-worker:loopback-video")
+
+	_ = submitAndAssert(t, td, payload, "completed", "completed")
+
+	logText := LogText(t)
+	if !strings.Contains(logText, "/dev/video0") {
+		t.Fatalf("Expected log to contain /dev/video0, but it didn't\n%s", logText)
+	}
+	if !strings.Contains(logText, "crw-rw----") {
+		t.Fatalf("Expected log to contain crw-rw----, but it didn't\n%s", logText)
+	}
+}
```

Comment:
This looks like a copy of the test below, but with a fixed value for the location. I don't think this test tests anything that the test below doesn't already test? Maybe we can delete it.

Reply:
So what I was trying to do by separating between different test cases was differentiate between the `strings.Contains(logText, "/dev/video0")` calls between the `ls -l /dev/video0` and `echo $TASKCLUSTER_VIDEO_DEVICE`. If one of those commands logged out `/dev/video0`, then the test would pass, and I wouldn't have successfully singled out which part actually was passing, the video device existing, or the environment variable existing.

Reply:
Ah that makes sense. Thanks. I think you can do this if when you echo it you have text, like `echo "Device: ${TASKCLUSTER_VIDEO_DEVICE}"` and then check for text `Device: /dev/video0`. I think `ls -l` will fail if the device does not exist, so then it should not be completed/completed, and I think test will fail already.
The test setup might set the device number to zero, but that lives in another file etc, that could get out of sync. Would it be worth adding a comment here that we can hardcode 0 because it is hardcoded in the setup method? Or maybe read it directly from the config here, so if it is changed in the setup, it is changed here too? (something like `fmt.Sprintf("/dev/video%v", config.LoopbackVideoDevice)`). Just want to make sure we don't have the zero hardcoded in multiple places, and if one changes, the other breaks. But e.g. a comment to say it has to match test setup is also ok, I think I just prefer reading the variable so the information isn't duplicated.